ios/Makefile

```diff
@@ -0,0 +1,9 @@
+WORKSPACE=DroidKaigi2021.xcworkspace
+
+.PHONY: open
+open:
+	@open $(WORKSPACE)
+
+.PHONY: bootstrap
+bootstrap:
+	mint bootstrap
```

Comment:
I though using SPM for manage XcodeGen is better. I want to know your idea using mint.

Reply:
`XcodeGen` means `SwiftLint` ..? 🤔 

No reasons for using mint in this PR 😢  
I will fix with using SPM for managing tools.